src/modules/Api.ts

```diff
@@ -675,6 +676,29 @@ export class Api {
     return response.data;
   }
 
+  public async getMarketV2({
+    market,
+  }: {
+    market: string,
+  }): Promise<{ market: ApiMarketMessageV2 }> {
+    const response = await axios({
+      url: `${this.endpoint}/v2/markets/${market}`,
+      method: 'get',
+      timeout: this.timeout,
+    });
+    return response.data;
+  }
+
+  public async getMarketsV2():
+    Promise<{ markets: { [market: string]: ApiMarketMessageV2 } }> {
```

Comment:
to return this type, we would have to parse the response strings into BigNumbers